### PR TITLE
[I18n] RBAC PUT permissions

### PR DESCRIPTION
**Pull request by soupette**
### What does it do?

It allows to the edit the permissions of a role.

### CODE REVIEW

File: packages/strapi-admin/admin/src/components/Roles/Permissions/utils/formatSettingsPermissionsToAPI.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+const formatSettingsPermissionsToAPI = settingsPermissionsObject => {
+  return Object.values(settingsPermissionsObject).reduce((acc, current) => {
+    const currentCategoryPermissions = createPermissionsArrayFromCategory(current);
+
+    return [...acc, ...currentCategoryPermissions];
+  }, []);
+};
+
+export default formatSettingsPermissionsToAPI;
+export { createConditionsArray, createPermission, createPermissionsArrayFromCategory };
```
Comment (HichamELBSI) [inline packages/strapi-admin/admin/src/components/Roles/Permissions/utils/formatSettingsPermissionsToAPI.js:53]: 👍🏽

File: packages/strapi-admin/admin/src/components/Roles/temp/fakeData.js [hunk 2]
```diff
@@ -398,34 +488,42 @@ const data = {
...
-          subjects: ['restaurant'],
-          applyToProperties: ['locales'],
-        },
+        // {
+        //   label: 'Publish',
+        //   actionId: 'plugins::content-manager.explorer.publish',
+        //   subjects: ['application::restaurant.restaurante'],
+        //   applyToProperties: [],
+        //   // applyToProperties: ['locales'],
+        // },
```
Comment (HichamELBSI) [inline packages/strapi-admin/admin/src/components/Roles/temp/fakeData.js:526]: Still need those comments ?

**Review by HichamELBSI — CHANGES_REQUESTED** [review]
Good job!

File: packages/strapi-admin/admin/src/components/Roles/temp/fakeData.js (hunk 2 above)
Comment (soupette) [reply packages/strapi-admin/admin/src/components/Roles/temp/fakeData.js:526]: Yes until we connect the view to the API, I'd rather keep the layout.

**Pull request merged**